GitUI/GitUIExtensions.cs

```diff
@@ -58,7 +58,7 @@ public static class GitUIExtensions
             {
                 if (!string.IsNullOrWhiteSpace(defaultText))
                 {
-                    return fileViewer.ViewTextAsync(item?.Item?.Name, defaultText);
+                    return fileViewer.ViewTextAsync(item.Item.Name, defaultText);
```

Comment:
@gerhardol this may cause NRE ❗

Reply:
Checked on line 57, not needed

Reply:
Here's the logic lines:57-61:

```
If `item == null OR item.Item == null OR item.SecondRevision == null OR item.SecondRevision.ObjectId == null` Then
  If `!string.IsNullOrWhiteSpace(defaultText)` Then
    return fileViewer.ViewTextAsync(item.Item.Name, defaultText);
```

So `item.Item.Name` can be `item == null OR item.Item == null` and thus result in NRE.